views/default/forms/admin/plugins/sort.php

```diff
@@ -8,7 +8,9 @@
  */
 
 echo '<div class="mtm">';
+echo '<label for="admin-plugins-sort" class="hidden">' . elgg_echo('sort') . '</label>';
```

Comment:
like with many things IE is lagging behind on support of the hidden attribute (source http://www.w3schools.com/tags/att_global_hidden.asp)
what is the support policy for browsers?